### PR TITLE
Reactivate-Service 

### PR DESCRIPTION
**Pull request by sumanvpacewisdom**
Changes done according to nov 3 , migrations , seeders , comments

### CODE REVIEW

File: src/migrations/20220722162957-notificationTemplates.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ let emailTemplates = [
 		subject: 'Your OTP to sign-up on MentorED',
 		body: '<div><p>Dear {name},</p> Your OTP to complete the registration process is {otp}. Please enter the OTP to complete the registration. For your security, please do not share this OTP with anyone.</div>',
 	},
+	{
+		code: 'reactivateotp',
```
Comment (ankitpws) [inline src/migrations/20220722162957-notificationTemplates.js:18]: This is not needed @sumanvpacewisdom

File: src/services/admin.js [hunk 2]
```diff
@@ -299,7 +299,7 @@ module.exports = class AdminHelper {
 					id,
 				},
 				{
-					status: common.inactiveStatus,
+					status: common.INACTIVE_STATUS,
```
Comment (ankitpws) [inline src/services/admin.js:302]: Why is This changed? @sumanvpacewisdom

File: src/constants/common.js [hunk 3]
```diff
@@ -82,10 +87,10 @@ module.exports = {
 	responseType: 'stream',
 	roleAssociationModel: 'UserRole',
 	roleAssociationName: 'user_roles',
-	activeStatus: 'ACTIVE',
-	inactiveStatus: 'INACTIVE',
-	roleMentor: 'mentor',
-	roleMentee: 'mentee',
+	ACTIVE_STATUS: 'ACTIVE',
```
Comment (ankitpws) [inline src/constants/common.js:90]: Why is this change needed? @sumanvpacewisdom

File: src/.env.sample [hunk 4]
```diff
@@ -155,3 +155,12 @@ MENTOR_REQUEST_ACCEPTED_EMAIL_TEMPLATE_CODE=mentor_request_accepted
 
 #Email template for mentor role request rejected
 MENTOR_REQUEST_REJECTED_EMAIL_TEMPLATE_CODE=mentor_request_rejected
+
+#Default role 
+DEFAULT_ROLE=mentee
```
Comment (ankitpws) [inline src/.env.sample:160]: Template code is missing for sending Email Template for OTP @sumanvpacewisdom

File: src/db/users/model.js [hunk 5]
```diff
@@ -77,6 +77,11 @@ const userSchema = new Schema(
 			type: String,
 			default: 'en',
 		},
+		status: {
+			type: String,
```
Comment (ankitpws) [inline src/db/users/model.js:81]: This is not needed @sumanvpacewisdom

Comment (ankitpws) [reply src/db/users/model.js:81]: its mongo code

File: src/.env.sample (hunk 4 above)
Comment (ankitpws) [reply src/.env.sample:160]: REACTIVATION_OTP_EMAIL_TEMPLATE_CODE

File: src/.env.sample [hunk 6]
```diff
@@ -155,3 +155,12 @@ MENTOR_REQUEST_ACCEPTED_EMAIL_TEMPLATE_CODE=mentor_request_accepted
 
 #Email template for mentor role request rejected
 MENTOR_REQUEST_REJECTED_EMAIL_TEMPLATE_CODE=mentor_request_rejected
+
+#Default role
```
Comment (ankitpws) [inline src/.env.sample:159]: Why this is added? @sumanvpacewisdom

**Review by ankitpws — CHANGES_REQUESTED** [review]
Reviewed on 14 Nov 2 Pm

File: src/migrations/20220722162957-notificationTemplates.js (hunk 1 above)
Comment (sumanvpacewisdom) [reply src/migrations/20220722162957-notificationTemplates.js:18]: @ankitpws then how do it take the notification template for reactivate otp

File: src/services/admin.js (hunk 2 above)
Comment (sumanvpacewisdom) [reply src/services/admin.js:302]: there was a code conflicts and they have added this to the new code and i needed to change to solve conflicts
@ankitpws

File: src/constants/common.js (hunk 3 above)
Comment (sumanvpacewisdom) [reply src/constants/common.js:90]: there was a code conflicts and they have added this to the new code and i needed to change to solve conflicts
@ankitpws

File: src/.env.sample (hunk 6 above)
Comment (sumanvpacewisdom) [reply src/.env.sample:159]: due to code conflicts, i wanted to change it @ankitpws
